.ci/checkTranslator.sh

```diff
@@ -123,8 +123,12 @@ usage () { (( $# > 0 )) && err "$*"; err "Usage: $0 <translator.js>"; }
 main() {
 
     # Add './node_modules/.bin to PATH for jsonlint
-    PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/node_modules/.bin:"$PATH"
+    PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/node_modules/.bin:"$PATH"
```

Comment:
No, it doesn't matter and is just cosmetic. Feel free to merge. (We are already planing the next commit which will check also the deletion of translators.)